docker-compose.yml

```diff
@@ -29,17 +27,33 @@ services:
     expose:
       - 5000
     ports:
-      - 5000:5000
-    command: ["postgraphile", "--connection", "postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DATABASE}", "--host", "0.0.0.0", "--port", "5000", "--schema", "trials_status_schema"]
+      - "${HOST_IP:-127.0.0.1}:${GRAPHQL_PORT}:${GRAPHQL_PORT}"
+    command:
+      [
```

Comment:
Did you intend to remove the `postgraphile` line here?

Reply:
Yep! I don't know why, but the current version of the postgraphile docker image changed the command to not need `postgraphile`.